airflow-core/src/airflow/api_fastapi/core_api/routes/public/xcom.py

```diff
@@ -69,41 +68,41 @@ def get_xcom_entry(
     stringify: Annotated[bool, Query()] = False,
 ) -> XComResponseNative | XComResponseString:
     """Get an XCom entry."""
-    if deserialize:
-        if not conf.getboolean("api", "enable_xcom_deserialize_support", fallback=False):
```

Comment:
Next step would be to wipe off this configuration

Reply:
Should this wiping out part also in 3.0.3?